types/README.md

```diff
@@ -14,44 +14,44 @@ const MY_FIELD_NAME = 'myField';
 
 
 export const extensions = (graphQL: GraphQL): Extensions => {
-	return {
-		types: {
-			[MY_OBJECT_TYPE_NAME]: {
-				description: 'Description for my object type',
-				fields: {
-					myString: {
-						type: graphQL.nonNull(graphQL.GraphQLString),
-					}
-				}
-			}
-		},
-		creationCallbacks: {
-			[ObjectTypeName.Content]: (params) => {
-				params.addFields({
-					[MY_FIELD_NAME]: {
-						type: graphQL.reference(MY_OBJECT_TYPE_NAME) // nullable
-					}
-				});
-			}
-		},
-		resolvers: {
-			[ObjectTypeName.Content]: {
-				[MY_FIELD_NAME]: () => {
-					const {
-						// args,
-						// localContext,
-						source: content
-					} = env;
-					if (content.type === 'portal:fragment') {
-						return null;
-					}
-					return {
-						myString: 'Hello, World!'
-					}
-				},
-			}
-		}
-	}
+  return {
+    types: {
+      [MY_OBJECT_TYPE_NAME]: {
+        description: 'Description for my object type',
+        fields: {
+          myString: {
+            type: graphQL.nonNull(graphQL.GraphQLString),
+          }
+        }
+      }
+    },
+    creationCallbacks: {
+      [ObjectTypeName.Content]: (params) => {
+        params.addFields({
+          [MY_FIELD_NAME]: {
+            type: graphQL.reference(MY_OBJECT_TYPE_NAME) // nullable
```

Comment:
type can not be nullable :)

Reply:
if type is null, then an error will be thrown in runtime

Reply:
Yah, I guess I meant optional. But I guess that's sorta all fields in GraphQL. So I'll just remove that confusing comment.